cli/cmd/resources/odiglet.go

```diff
@@ -184,6 +184,15 @@ func NewOdigletClusterRole(psp bool) *rbacv1.ClusterRole {
 					"daemonsets/finalizers",
 				},
 			},
+			{
+				Verbs: []string{
+					"get",
+				},
+				APIGroups: []string{"apps"},
+				Resources: []string{
+					"replicasets",
+				},
```

Comment:
Does this also imply that Odiglet will now hold all the ReplicaSets in the cache? That used to be the case and when it was removed it saved a lot of memory.

Reply:
Notice that the verbs for this rule is only "get", and controller-runtime cache requires a "watch" and "list", thus even if someone uses it with `client.Client`, it will not work.